src/pyedb/dotnet/edb_core/stackup.py

```diff
@@ -68,16 +68,161 @@
 logger = logging.getLogger(__name__)
 
 
-class Stackup(object):
+class LayerCollection(object):
+    AUTO_REFRESH = True
```

Comment:
```suggestion
    auto_refresh = True
```
capslock should be used for constant values (here this attribute might be changed by the user)